docs/advanced/singlenodeclusters.md

```diff
@@ -0,0 +1,33 @@
+---
+id: singlenodeclusters
+sidebar_position: 7
+sidebar_label: Single-Node Clusters
+title: "Single-Node Clusters"
+keywords:
+- Single Node
+Description: Support for deployment of single-node clusters
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.2/advanced/singlenodeclusters">
+</head>
+
+Beginning with v1.2.0, Harvester is supporting single-node clusters for implementations that require minimal initial deployment resources and/or that can tolerate lower resiliency. You can create single-node clusters using the standard installation methods ([ISO](https://docs.harvesterhci.io/v1.3/install/index), [USB](https://docs.harvesterhci.io/v1.3/install/usb-install), and [PXE boot](https://docs.harvesterhci.io/v1.3/install/pxe-boot-install)).
+
+Single-node clusters support most Harvester features, including the creation of RKE2 clusters and node upgrades (with some limitations). However, this deployment type has the following key disadvantages:
+
+- No high availability: Errors and updates that require rebooting of the node cause downtime to running VMs.
+- No multi-replica support: Only one replica is created for each volume in Longhorn.
+- No live migration and zero-downtime support during upgrades
+
+## Prerequisites/Requirements
```

Comment:
Do we usually use the slash here? I'd suggest just using "Prerequisites"

Reply:
I'll use "Prerequisites" as suggested.